packages/fetch-router/src/lib/route-handlers.ts

```diff
@@ -32,27 +32,28 @@ export function isRouteHandlersWithMiddleware<T extends RouteMap>(
 /**
  * An individual route handler.
  */
-export type RouteHandler<M extends RequestMethod | 'ANY', P extends string> =
-  | RequestHandlerWithMiddleware<M, P>
-  | RequestHandler<M, Params<P>>
+export type RouteHandler<method extends RequestMethod | 'ANY', pattern extends string> =
+  | RequestHandlerWithMiddleware<method, pattern>
+  | RequestHandler<method, Params<pattern>>
 
-type RequestHandlerWithMiddleware<M extends RequestMethod | 'ANY', P extends string> = {
-  use: Middleware<M, Params<P>>[]
-  handler: RequestHandler<M, Params<P>>
+type RequestHandlerWithMiddleware<method extends RequestMethod | 'ANY', pattern extends string> = {
+  use: Middleware<method, Params<pattern>>[]
+  handler: RequestHandler<method, Params<pattern>>
 }
 
-export function isRequestHandlerWithMiddleware<M extends RequestMethod | 'ANY', P extends string>(
-  handler: any,
-): handler is RequestHandlerWithMiddleware<M, P> {
+export function isRequestHandlerWithMiddleware<
+  method extends RequestMethod | 'ANY',
+  pattern extends string,
+>(handler: unknown): handler is RequestHandlerWithMiddleware<method, pattern> {
```

Comment:
```suggestion
export function isRequestHandlerWithMiddleware<
  Method extends RequestMethod | 'ANY',
  Pattern extends string,
>(handler: unknown): handler is RequestHandlerWithMiddleware<Method, Pattern> {
```

Reply:
This is an example where it would be quite easy to accidentally use `RequestMethod` instead of `Method` in the return type (i.e. `handler is RequestHandlerWithMiddleware<RequestMethod, Pattern>`), whereas the lowercase parameter names makes it much clearer what should be passed into the generic.